src/Vendr.PaymentProviders.Worldpay/Helpers/Md5Helper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+
+namespace Vendr.PaymentProviders.Worldpay.Helpers
+{
+    public static class Md5Helper
+    {
+        public static string CreateMd5(string input)
+        {
+            return new MD5CryptoServiceProvider().ComputeHash(Encoding.UTF8.GetBytes(input)).ToHex(true);
+        }
+
+        public static string ToHex(this byte[] bytes, bool toLower)
+        {
+            var c = new char[bytes.Length * 2];
+            for (int bx = 0, cx = 0; bx < bytes.Length; ++bx, ++cx)
+            {
+                var b = ((byte)(bytes[bx] >> 4));
+                c[cx] = (char)(b > 9 ? b + 0x37 + 0x20 : b + 0x30);
+
+                b = ((byte)(bytes[bx] & 0x0F));
+                c[++cx] = (char)(b > 9 ? b + 0x37 + 0x20 : b + 0x30);
+            }
+
+            return toLower ? new string(c).ToLower() : new string(c);
+        }
```

Comment:
@mattbrailsford it could be useful with a method in Vendr `PaymentProviderBase` class to hex encode.
I have used this in Reepay: https://github.com/vendrcontrib/vendr-payment-provider-reepay/blob/5bfee0552d1e0e825170d238cadc97112705648b/src/Vendr.Contrib.PaymentProviders.Reepay/ReepayPaymentProviderBase.cs#L226-L229